app/components/operator-form/component.js

```diff
@@ -1,35 +1,12 @@
 import Ember from 'ember';
 
 export default Ember.Component.extend({
-	timezones: [
-		"America/New_York",
-		 "America/Detroit",
-		 "America/Kentucky/Louisville",
-		 "America/Kentucky/Monticello",
-		 "America/Indiana/Indianapolis",
-		 "America/Indiana/Vincennes",
-		 "America/Indiana/Winamac",
-		 "America/Indiana/Marengo",
-		 "America/Indiana/Petersburg",
-		 "America/Indiana/Vevay",
-		 "America/Chicago",
-		 "America/Indiana/Tell_City",
-		 "America/Indiana/Knox",
-		 "America/Menominee",
-		 "America/North_Dakota/Center",
-		 "America/North_Dakota/New_Salem",
-		 "America/North_Dakota/Beulah",
-		 "America/Denver",
-		 "America/Boise",
-		 "America/Phoenix",
-		 "America/Los_Angeles",
-		 "America/Metlakatla",
-		 "America/Anchorage",
-		 "America/Juneau",
-		 "America/Sitka",
-		 "America/Yakutat",
-		 "America/Nome",
-		 "America/Adak",
-		 "Pacific/Honolulu"
-	]
+	actions: {
+		clickYes() {
+			this.set('operator.include_in_changeset', true);
+		},
+		clickNo() {
```

Comment:
'concise methods' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).